src/main/java/com/minecolonies/coremod/sounds/AbstractWorkerSounds.java

```diff
@@ -0,0 +1,68 @@
+package com.minecolonies.coremod.sounds;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * I worker sounds interface which contains all methods worker sounds should have.
+ */
+public abstract class AbstractWorkerSounds
+{
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 25;
+
+    /**
+     * Chance to play a basic sound.
+     */
+    private static final int BASIC_SOUND_CHANCE = 100;
+
+    /**
+     * List of sound handlers.
+     */
+    public static final List<AbstractWorkerSounds> handlers = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2386)